### PR TITLE
10.0 mig sales team operating unit

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Sales Team Operating Unit
=========================

This module introduces the following features:

* Adds the Operating Unit (OU) to the Sales Team.

* The user's default Operating Unit (OU) is proposed at the time of creating
  the Sales Team.

* Security rules are defined to ensure that users can only see the Sales Team
  of that Operating Units in which they are allowed access to.

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
depends on https://github.com/OCA/operating-unit/pull/45

File: sales_team_operating_unit/models/crm_team.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# Copyright 2017-TODAY Serpent Consulting Services Pvt. Ltd.
+#   (<http://www.serpentcs.com>)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
```
Comment (serpentcs-dev1) [inline sales_team_operating_unit/models/crm_team.py:9]: @lreficent unused, it is blocking flake8 can you please remove it?

**Review by serpentcs-dev1 — CHANGES_REQUESTED** [review]

File: sales_team_operating_unit/models/crm_team.py (hunk 1 above)
Comment (LoisRForgeFlow) [reply sales_team_operating_unit/models/crm_team.py:9]: yes, sure

Comment (LoisRForgeFlow) [reply sales_team_operating_unit/models/crm_team.py:9]: It wasn't in that file. But yes, it should be ok now.

Comment (serpentcs-dev1) [reply sales_team_operating_unit/models/crm_team.py:9]: @lreficent Sorry my mistake it has to removed from test cases but instead we removed from crm_team.py

So you need to add the import again to py file and remove it from test cases.

Comment (LoisRForgeFlow) [reply sales_team_operating_unit/models/crm_team.py:9]: solved.

**Comment by LoisRForgeFlow** [comment]
This seems fine. Can you review? @jbeficent @aheficent @mreficent

**Review by serpentcs-dev1 — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
:+1:

**Review by JordiBForgeFlow — APPROVED** [review]

**Pull request merged**